entries/z/zomato.com.json

```
@@ -6,8 +6,33 @@
       "facebook": "zomato",
       "twitter": "Zomato"
     },
+    "regions": [
+      "ae",
+      "au",
+      "br",
+      "ca",
+      "cl",
+      "cz",
+      "gb",
+      "in",
+      "ie",
+      "it",
+      "lb",
+      "lk",
+      "my",
+      "nz",
+      "ph",
+      "pl",
+      "pt",
+      "qa",
+      "sg",
+      "sl",
+      "za",
+      "tr",
+      "us"
```

```suggestion
      "ae",
      "au",
      "br",
      "ca",
      "cl",
      "cz",
      "gb",
      "in",
      "id",
      "ie",
      "it",
      "lb",
      "lk",
      "my",
      "nz",
      "ph",
      "pl",
      "pt",
      "qa",
      "sg",
      "sk",
      "sl",
      "za",
      "tr",
      "us",
      "za"
```
There were a few more regions on the country selector dropdown